route/rwby-router.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const debug = require('debug')('rwby:router');
+const Rwby = require('../model/rwby.js');
+const rwbyRouter = new Router();
+
+rwbyRouter.post('/api/rwby', jsonParser, function(req, res, next){
+  debug('post api/rwby');
+
+  Rwby.createRwby(req.body)
+  .then( rwby => res.json(rwby))
+  .catch( err => next(err));
+});
+
+rwbyRouter.get('/api/rwby/:id', function(req, res, next){
+  debug('get api/rwby/:id');
+
+  Rwby.fetchRwby(req.params.id)
+  .then( rwby => res.json(rwby))
+  .catch( err => next(err));
+});
+
+rwbyRouter.put('/api/rwby/:id', function(req, res, next){
```

Comment:
I'm sure you understand why we need this. If not, we can talk about it tomorrow, but in the meantime, this should also fix your post test.